lambdas/db-migration/src/migrations/20201014105030_create_async_operations_table.ts

```diff
@@ -23,6 +23,7 @@ export const up = async (knex: Knex): Promise<void> =>
           'Kinesis Replay',
           'Reconciliation Report',
           'Migration Count Report',
+          'Data Migration 2',
```

Comment:
Were you able to run an async operation using this as the operation type?

Reply:
For the record, yes I was able to run it using the new operation type